_posts/2021-04-09-prace-workshop.md

```diff
@@ -26,4 +26,13 @@ The workshop is open to everyone, but places are limited. Apply early!
 
 For more information and the registration form, please see the [workshop overview](https://events.prace-ri.eu/event/1201/overview) website.
 
-- @micaela-matta, @IAlibay, @orbeckst, @jbarnoud, @fiona-naughton, @lilyminium, @mnmelo, @richardjgowers (the workshop organising committee)
+## Workshop materials
+
+All workshop materials are publicly available in the repository [MDAnalysis/WorkshopPrace2021](https://github.com/MDAnalysis/WorkshopPrace2021). 
+You can run the Jupyter notebooks locally on your own computer.
+All materials use the *beta version of MDAnalysis 2.0.0*, which you can easily install by following the [installation instructions for 2.0.0b](https://github.com/MDAnalysis/WorkshopPrace2021/blob/main/INSTALL.md).
```

Comment:
This is a little bit misleading. I'd instead just say this a link to the install instructions for the whole workshop and mention as a note that it requires the beta version of 2.0.0?

Reply:
fair enough, would be happy to change

(At the moment these are the cleanest written instructions for 2.0.0b that we have and I send them to anyone who wants to try the beta because I know that you get a working installation, including OpenMM, RDKit, HDF5 etc.)